pkg/META.in

```diff
@@ -1,19 +0,0 @@
-version = "%{version}%"
-description = "[@@deriving yojson]"
-requires(-ppx_driver) = "ppx_deriving ppx_deriving_yojson.runtime"
-ppxopt(-ppx_driver,-custom_ppx) = "ppx_deriving,./ppx_deriving_yojson.cma"
-requires(ppx_driver) = "ppx_deriving.api"
-archive(ppx_driver, byte) = "ppx_deriving_yojson.cma"
-archive(ppx_driver, native) = "ppx_deriving_yojson.cmxa"
```

Comment:
Does jbuilder correctly generate all the ppx_driver clauses?

Reply:
yes